src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorRequestsServiceImpl.java

```diff
@@ -281,6 +278,16 @@ private boolean isoCountryCodeTranslatesToSingleFipsCountryCode(String isoCountr
         }
     }
     
+    private boolean bankIsoCountryCodeTranslatesToSingleFipsCountryCodeWhenBankDataIsEntered(String isoCountryCode, List<String> errorMessages) {
+        if (ObjectUtils.isNotNull(isoCountryCode)) {
+            return isoCountryCodeTranslatesToSingleFipsCountryCode(isoCountryCode, errorMessages);
+        }
+        else {
```

Comment:
Another "else" block start that could use adjustment.

Reply:
Moved.